inst/include/component_data.hpp

```diff
@@ -326,14 +326,13 @@
 #define D_CO3_LL				"CO3_LL"
 #define D_CO3_HL				"CO3_HL"
 #define D_CO3				    "CO3"
-#define D_ATM_OCEAN_CONSTRAIN   "atm_ocean_constrain"
 #define D_TIMESTEPS             "ocean_timesteps"
 #define D_REVELLE_HL            "Revelle_HL"
 #define D_REVELLE_LL            "Revelle_LL"
 
 // SimpleNbox component
 #define D_NBP                   "NBP"
-#define D_ATMOSPHERIC_CO2       "Ca"
+#define D_ATMOSPHERIC_CO2       "CO2_conc"
 #define D_PREINDUSTRIAL_CO2     "C0"
 #define D_ATMOSPHERIC_C         "atmos_c"
```

Comment:
@bpbond should this be atmos_co2? Since it is atmos CO2 in units of C?

Reply:
It's not in units of C; it's parts per million (vol).

>we discussed switching to atmos_co2 what about CO2_conc to be consistent with all of the other ghg concentration naming patterns?

Hmm, both have advantages. I think `CO2_conc` makes the most sense?

Reply:
I think CO2_conc makes the most sense. But I was wondering if D_ATMOSPHERIC_C should go from atmos_c to atmos_co2, because that is less misleading since D_ATMOSPHERIC_C does not include carbon from non co2 sources?

Reply:
`atmos_co2` seems highly misleading to me; the variable is tracking carbon, not CO2. 

`atmos_co2-c`?